### PR TITLE
chore: Configure eslint

### LINKED ISSUE #4: Configure eslint
### Description

_No response_

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by shivamvishwakarm**
### Description

Configure eslint for code consistency for our project.

### Checklist:

- [ ✓] This pull request follow our contribution guidelines
- [ ✓] I have performed a self-review of my own code
- [✓ ] I have commented my code, particularly in hard-to-understand areas
- [ ✓] I have made corresponding changes to the documentation (postman and swagger)
- [✓ ] I have written tests for the code
- [✓ ] I have updated .env.example file for new .env variables

### What is current behavior?

Closes #4 

### What is new behavior?

Now the code is more consistent and code is written under some rules define under .eslintrc.js

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -47,8 +47,12 @@
 		"@types/jsonwebtoken": "8.5.9",
 		"@types/passport": "1.0.11",
 		"@types/passport-google-oauth20": "2.0.11",
+		"@typescript-eslint/eslint-plugin": "^5.38.1",
+		"@typescript-eslint/parser": "^5.38.1",
 		"add": "2.0.6",
 		"all-contributors-cli": "6.22.0",
+		"eslint": "^8.24.0",
+		"eslint-plugin-react": "^7.31.8",
```
Comment (aayushchugh) [inline package.json]: remove react plugin because we don't need it in backend

**Review by aayushchugh — CHANGES_REQUESTED** [review]
please create a `.prettierrc` for prettier config

```json
{
	"semi": true,
	"printWidth": 100,
	"singleQuote": false,
	"trailingComma": "all",
	"jsxSingleQuote": false,
	"bracketSpacing": true,
	"tabWidth": 4,
	"useTabs": true,
	"endOfLine": "lf",
	"extends": ["plugin:react/recommended", "prettier"]
}
```

File: package.json (hunk 1 above)
Comment (shivamvishwakarm) [reply package.json]: ok

**Review by aayushchugh — APPROVED** [review]
Nice work @shivamvishwakarm 👍🏼

**Comment by shivamvishwakarm** [comment]
Thank's @aayushchugh

**Comment by aayushchugh** [comment]
@shivamvishwakarm for marking checkbox as checked you can do like this

`[x]` instead of `[ ✓]`
- [x] This pull request follow our contribution guidelines

**Review by BitOfAByte — APPROVED** [review]

**Pull request merged**